### DISCUSSION TITLE
How to test BigNumbers/BN ?

### DISCUSSION THREAD

**Discussion by xluckydegen** [ConsenSys-archive/truffle#5742] [General Q&A]
Hi,
I just learning the truffle but I'm not able to find a way how to use chai also for big numbers.

I found `chain-BN` but I don't know how to integrate it with truffle.

```
  it('BN test', async () =>
  {
    const b1= new BN(1);
    const b2 = new BN(1);
    assert.equal(b1, b2, "should be equal");
  });
```

with result

```
       BN test:

      should be equal
      + expected - actual


      at Context.<anonymous> (test\coin.test.ts:11:12)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

Any advice, please?

**Comment by apdan510** [comment]
@xluckydegen could you open a support ticket with us? It will be easier to share and take a close at your repo together.

https://trufflesuite.zendesk.com/hc/en-us/requests/new

Kind regards,
Alex | Truffle Support | Consensys

**Comment by xluckydegen** [comment]
Thank you for the reply. Unfortunately, I decided to use HardHat after these issues so I don't have these test projects anymore.
